README.md

```diff
@@ -202,6 +202,14 @@ Once installed set the files as such:
 Then to make sure prettier is run correctly, modify the on save actions:
 ![Intellij javascript onsave](intellij/intellij-javscript-actionsonsave.png)
 
+For Scala:
+
+Go to `Preferences/Settings > Editor > Code Style > Scala > Imports` and update:
+* Class count to use import with '_' -> 999
+* Ensure `Merge imports with the same prefix into one statement` is checked
+
+Then go to `Preferences/Settings > Editor > General > Auto import` and uncheck "Optimize imports on the fly" for Scala. This is necessary because there are some imports that are not explicitly used in the code but are necessary for the code to compile.
```

Comment:
I'm wondering if this should be in the common configuration.
I think the behavior we face in Armor (import that looks unused while required for the compilation) is a bit specific, it should not impact any Scala project.
What do you think about moving this sentence to the Armor Readme instead?

Reply:
Fair point. I was not aware of this and assumed that was a common Scala scenario. I agree with you then. I will remove it from here, and you can maybe add it in the readme while you are on your ticket. Or if you think it does not belong there, I can add it (now or once you are done with your ticket)

Reply:
I will add it in the context of my ticket.